wasm/TSF/build.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
+
+set -euo pipefail
+
```

Comment:
WDYT about adding some log file with a last build timestamp and Emscripten version for reproducibility? Something like
```
BUILD_LOG="$(realpath build.log)"
echo -e "Built on $(date --rfc-3339=seconds)\n" | tee "$BUILD_LOG"

echo "Toolchain versions" | tee -a "$BUILD_LOG"
emcc --version | head -n1 | tee -a "$BUILD_LOG"
```
(shameless copy from [sqlite3/build.sh](https://github.com/WebKit/JetStream/pull/2/files#diff-e7f726f906491117444ff9d200e03b79188411304fd391ec888a420f1d7ffbd2))

Reply:
Yeah, I was thinking we should just have one script to do it rather than duplicating the logic. That said, I can do that in a follow up https://github.com/WebKit/JetStream/issues/22. I'll copy your logic for now.

Reply:
It seems like `date --rfc-3339=seconds` isn't supported by MacOS's out of the box date. I'm gonna go with `date -u '+%Y-%m-%dT%H:%M:%SZ'`

Reply:
Ah, good to know and keep in mind for an eventual (shared?) build script.